lib/newman_conway.py

```diff
@@ -7,4 +7,18 @@ def newman_conway(num):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    if num == 0:
+        raise ValueError
+
+    arr  = [0,1,1]
+    for i in range(3,num + 1):
+        r = arr[arr[i-1]]+arr[i-arr[i-1]] 
+        arr.append(r)
+    
+    string_array = []
+    for num in arr[1:]:
+        string_array.append(str(num))
+    string_array = ' '.join(string_array)
+        
+    return string_array
```

Comment:
⚠️ 
This doesn't work, you instead need an array to keep track of prior newman conway numbers.